server/pkg/schema/type_property.go

```diff
@@ -230,6 +230,59 @@ func (t *TypeProperty) Match(m TypePropertyMatch) {
 	}
 }
 
+func (t *TypeProperty) UpdateFrom(t2 *TypeProperty) {
```

Comment:
Since the processing is almost the same as for `Clone` method, it is a duplicate code and should be removed.